.eslintrc.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
```

Comment:
this file is taken directly from [typescript-action's version](https://github.com/actions/typescript-action/blob/9cdc5b3/.eslintrc.json) with few rules tweaked (semi).

Reply:
:clap: :medal_military: